GitUI/Editor/RichTextBoxXhtmlSupportExtension.cs

```diff
@@ -1076,7 +1076,11 @@ public static string GetPlainText(this RichTextBox rtb, int from, int to)
                 {
                     // select one character
                     rtb.Select(i, 1);
-                    text.Append(rtb.SelectedText);
+                    string selectedText = rtb.SelectedText;
+                    if (!selectedText.StartsWith(LinkSeparator))
```

Comment:
LinkSeparator is 3 bytes and only one is selected, how can this work?
Comment?

Reply:
I have added a comment.

Reply:
I would have preferred to see the comment on line 1077, now I have to read back and forward
```
//  Try to select one visible character (RichTextBox returns a string, may contain more than what is visible)
```
But approving a merge